models/app.model.js

```diff
@@ -0,0 +1,59 @@
+const db = require("../db/connection");
+
+exports.selectTopics = () => {
+  return db
+    .query("SELECT slug, description FROM topics;")
+    .then((res) => {
+      return res.rows;
+    })
+    .catch((err) => {
+      console.log(err);
+    });
+};
+
+exports.selectArticleId = (article_id) => {
+  return db
+    .query(
+      "SELECT author, title, article_id, body, topic, created_at, votes FROM articles WHERE article_id = $1;",
+      [article_id]
+    )
+    .then((res) => {
+      return res.rows;
+    })
+    .catch((err) => {
+      console.log(er);
+    });
+};
+
+exports.selectArticleComment = (article_id) => {
+  return db
+    .query("SELECT comment_count FROM", [article_id])
```

Comment:
I think we spoke about this briefly, but take a look at using the relevant JOIN here to allow you to count the comments with the relevant article_id.